src/engine/routing_algorithms/routing_base_ch.cpp

```diff
@@ -139,7 +139,124 @@ void search(const datafacade::ContiguousInternalMemoryDataFacade<algorithm::CH>
     }
 }
 
-// assumes that heaps are already setup correctly.
+double getPathDistance(const datafacade::ContiguousInternalMemoryDataFacade<Algorithm> &facade,
```

Comment:
Why does this function need to be duplicated? Is there a way to share the implementation with the MLD variation?